simpleclient_hdrhistogram/src/main/java/io/prometheus/client/HdrSummary.java

```diff
@@ -92,6 +94,7 @@ private HdrSummary(Builder b) {
   public static class Builder extends SimpleCollector.Builder<Builder, HdrSummary> {
 
     private final List<Double> quantiles = new ArrayList<Double>();
+    private long highestToLowestValueRatio = 1000000;
```

Comment:
I increased the default initial dynamic range from `2` to `1000000` to avoid blocking caused by `auto-resizing` the newest bucket after rotation.

https://github.com/HdrHistogram/HdrHistogram/blob/master/src/main/java/org/HdrHistogram/ConcurrentDoubleHistogram.java